test/functional/info/info-basic.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
```

Comment:
you don't need to change anything in this PR, but for future reference, if you don't define any test_setup() a default one is run by testlib, and the default one looks like the one you created, it just creates a simple test environment. Same with the test_teardown(), so you could have gotten away with not adding them to this test.

Reply:
I need a to create an environment with a format other than staging. Looks like the teardown wasn't needed.  But will keep it for clarity.

Reply:
ahh I see, then yes, you needed that 😄